### ISSUE TITLE
Assert failed

### ISSUE THREAD

**Issue by RonivanFontanez** [peterhaneve/ONIMods#99]
After implement the PLib in my mods, I found this strange error during the game test. 

Assert failed: First anim file for CozyBedComplete needs to be non-null.

There are no errors in the VS during development, this only happens at the game test. 

It happens both in DLC and the Vanilla base game. Before implementing your lib, it worked completely fine.

**Comment by peterhaneve** [comment]
Can you include a full output log?

**Comment by RonivanFontanez** [comment]
This is the complete player.log: https://drive.google.com/file/d/1zMWv9VDcc-xeq-YLkjc8z5NxQKpgyNfg/view?usp=sharing

This is the short log provided during the blackhole: https://drive.google.com/file/d/1-uqdafUWpqzpQ8XgwQHaCP8Vi7SWR8yT/view?usp=sharing

**Comment by RonivanFontanez** [comment]
Today I did several tests trying to track down the source of the error, with no success. This log was produced during such tests, with no other mod activated except the one I was testing.

https://drive.google.com/file/d/1yNXDmT4AOX0XTW6K_uu4BrID7jFFBmuy/view?usp=sharing

I don't quite understand such issue, its like the game was not able to ''see'' the animation files, and consider it to be absent.

**Comment by peterhaneve** [comment]
```System.Reflection.TargetException: Non-static field requires a target
  at System.Reflection.MonoField.GetValue (System.Object obj) [0x0000b] in <9577ac7a62ef43179789031239ba8798>:0 
  at LocString.CreateLocStringKeys (System.Type type, System.String parent_path) [0x0006a] in C:\jenkins_workspace\workspace\SimGame_Windows\game\Assets\scripts\game\LocString.cs:85 
  at Localization.RegisterForTranslation (System.Type locstring_tree_root) [0x00043] in C:\jenkins_workspace\workspace\SimGame_Windows\game\Assets\scripts\game\Localization.cs:575
  ```

The first crash in your log is actually related to localization. It appears that the class you are registering for PLib localization has a `LocString` field which is not static, which crashes in the Klei `LocString` APIs. Make sure that your localization classes are properly formed.

**Comment by RonivanFontanez** [comment]
I don't have the slightest idea why its having an issue with non-static class declaration, since I followed the guideline using all static fields. I made a compacted, separated version of this mod, with only one buildings class, the buildings_patch and the buildings_strings just to be sure I was not missing anything. Seems by the log its saying that the buildings_patch.onLoad is having this issue, but everything is declared static on it. Believe me, I remade the entire thing from the scratch, and it still is giving the same error.

**Comment by RonivanFontanez** [comment]
I took your advice and rebuild the entire thing from the scratch. This time I made a separated mod, with a single building class, the buldings_patch and the buildings_string. I took my time to specifically set each field correctly, same as the guidelines, and still going with the same error. What amazes me is the fact your PLib worked perfectly in the first mod I updated with it, the Spacer Set. Its currently released on Steam and is working perfectly. I used exactly the same guidelines I used with it in this furniture mod, but it don't work. I lost the count of times I check each and very class look for any discrepancy, and there is absolutely none. Instead of coffee, the only thing I want to taste now is a cold pipe of a shotgun in my mouth.
